src/main/java/com/kuba6000/mobsinfo/config/Config.java

```diff
@@ -88,9 +92,20 @@ private static void load(Configuration configuration) {
                         _CacheRegenerationTrigger.ModAdditionRemovalChange.ordinal(),
                         c.toString())
                     .getInt());
+
             includeEmptyMobs = configuration
                 .get(category.get(), "IncludeEmptyMobs", true, "Include mobs that have no drops in NEI")
                 .getBoolean();
+
+            mobTimeout = configuration
+                .get(
+                    category.get(),
+                    "MobTimeout",
+                    10d,
```

Comment:
Is this time enough to not change anything in GTNH?

Reply:
It depends on your machine and if you get unlucky. If your machine is really slow, this might skip a lot of mobs - on my machine it skips most SpecialMobs witches, as they take forever to process. I can increase the default or remove it entirely if you want - 10s/mob is just the longest I would be willing to wait.

Reply:
It only skips Twilight Lich for me, but I guess we can't do anything about it.
The only problem I see here is that clients could have diffrent drops in NEI than it is on the server because the server doesn't sync the drops currently. But it is a problem outside of this pr